### PR TITLE
Add Django 5.0 support

### PR DESCRIPTION
**Pull request by pfouque**
Add Django 5.0 support to the test matrix and package description

Type: Feature

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+    py{38,39,310,311}-django42-psycopg2
+    py{38,39,310,311}-django41-psycopg2
+    py{38,39,310}-django32-psycopg2
+    report
 
 [testenv]
 install_command = pip install {opts} --no-compile {packages}
 deps =
-    django32: Django>=3.2,<3.3
+    django50: Django==5.0rc1
```
Comment (pfouque) [inline tox.ini:15]:
```suggestion
    django50: Django>=5.0rc1,<5.1
```

**Comment by wesleykendall** [comment]
Thanks @pfouque !

I do upgrades across all of the opus10 libraries at once by updating the template at https://github.com/Opus10/public-django-app-template and then propagating all of the changes with https://github.com/opus10/footing

In other words, the django 5 upgrade will be automated, but I will still be sure to give you a co-authored-by credit in the commit when it happens in the coming days

Thanks for your help!

**Comment by wesleykendall** [comment]
Changes went out in this pr https://github.com/Opus10/django-pgtrigger/pull/144

Added you as a co author and to readme. Thanks again!

**Comment by pfouque** [comment]
Thanks!

**Pull request closed without merging**